### ISSUE TITLE
Retrieving type/format/min/mag/etc. information from FBO/texture

### ISSUE THREAD

**Issue by realazthat** [mikolalysenko/regl#273]
Would be nice if one could reliably do:

```
if (texture.format === 'rgb') {
  // ...
}
```

Similarly for textures and fbos:

```
if (texture.type === 'uint8') {
  // ...
}
```

**Comment by Erkaman** [comment]
I could work on adding this, but I first want to hear if @mikolalysenko has any objections?

**Comment by Erkaman** [comment]
So I will work on adding this feature.

**Comment by Erkaman** [comment]
Resolved with PR #276.
